config/config-istio.yaml

```diff
@@ -44,6 +44,11 @@ data:
     # used pre-0.3.
     gateway.knative-ingress-gateway: "istio-ingressgateway.istio-system.svc.cluster.local"
 
+    # The default namespace of the gateway is the same with the serving
```

Comment:
This is a kind of expansion of `gateway.knative-ingress-gateway`, so should we combine L#42-45 with L47-50? I _personally_ think that we can remove `gateway.knative-ingress-gateway` from the example. (We can remove it from the code for compatibility, though.) WDYT?

Reply:
Maybe it is still meaningful to clarify that the default gateway is `gateway.knative-ingress-gateway` or 
 `gateway.knative-serving.knative-ingress-gateway`?